iml-system-test-utils/src/vagrant.rs

```diff
@@ -410,21 +439,74 @@ pub async fn add_docker_servers<S: std::hash::BuildHasher>(
         .await
 }
 
+pub async fn configure_rpm_setup(
+    setup: &SetupConfigType,
+    cluster_config: &ClusterConfig,
+) -> Result<(), CmdError> {
+    let setup_config: &SetupConfig = setup.into();
+
+    if setup_config.use_stratagem {
+        ssh::create_iml_config_dir(cluster_config.manager_ip).await?;
+
+        ssh::ssh_exec(
+            cluster_config.manager_ip,
+            format!(
+                r#"cat <<EOF > {}
+{}
+EOF
+"#,
+                format!("{}/stratagem-server.profile", iml::IML_RPM_CONFIG_PATH),
+                STRATAGEM_SERVER_PROFILE,
+            )
+            .as_str(),
+        )
+        .await?;
+
+        ssh::ssh_exec(
+            cluster_config.manager_ip,
+            format!(
+                r#"cat <<EOF > {}
+{}
+EOF
+"#,
+                format!("{}/stratagem-client.profile", iml::IML_RPM_CONFIG_PATH),
+                STRATAGEM_CLIENT_PROFILE,
+            )
+            .as_str(),
+        )
+        .await?;
+
+        run_vm_command(
+            cluster_config.manager,
+            format!(
+                "sudo chroma-config profile register {}/stratagem-server.profile \
```

Comment:
We can use `iml server profile load` now. It would read from stdin and I was thinking that in other places we could transmit from the host to VM via pipes, like `echo ... | ssh 'cat > /destination'`.

Reply:
I'll update to use `iml server profile load`